plugin/src/main/cpp/export/pico_export_plugin.cpp

```diff
@@ -200,5 +205,25 @@ String PicoEditorExportPlugin::_get_android_manifest_application_element_content
 		contents += "        <meta-data tools:node=\"replace\" android:name=\"Hand_Tracking_HighFrequency\" android:value=\"1\" />\n";
 	}
 
+	HybridType hybrid_type = _get_hybrid_app_setting_value();
+	if (hybrid_type != HYBRID_TYPE_DISABLED) {
+		contents += "        <meta-data tools:node=\"replace\" android:name=\"pvr.app.type\" android:value=\"2d\" />\n";
```

Comment:
I'm curious about this meta-data.. should its value be `vr` when we set the immersive activity as the default launcher?

Reply:
I think it might just control where the app is sorted in the library in headset. I need to experiment with that a bit.

But if that is correct, then, yeah, perhaps we should only set it when we start as panel, with the idea that the app is primarily a panel app.